src/main/java/com/tictactoe/SelectedGameSettings.java

```diff
@@ -0,0 +1,9 @@
+package com.tictactoe;
+
+public class SelectedGameSettings {
```

Comment:
It is kind of weird that you have `GameSettings` and `SelectedGameSettings`. Right now, I don't think you have to change this. However, if you get a chance, think about how your overall architecture could be simplified if you were to write this app again from scratch.